MCprep_addon/materials/prep.py

```diff
@@ -119,6 +119,11 @@ def pack_formats(self, context):
 		description="Change the pack format when using a PBR resource pack.",
 		items=pack_formats
 	)
+	useEmission = bpy.props.BoolProperty(
+		name="Use Emission",
+		description="Make emmisive materials emit light",
```

Comment:
Is it accurate (as of this PR) to say this is only relevant for non-simple prep? Might add that to the description.

Reply:
This applies to simple materials as they also emit light, just differently (using the principled shader instead of a separate node setup)